Bookish/Bookish.Web/Controllers/HomeController.cs

```diff
@@ -29,6 +29,33 @@ public IActionResult BookPage()
             List<BookInfo> books = (List<BookInfo>)db.Query<BookInfo>("select * from BookInfo");
             return View(books);
         }
+        public IActionResult BookInfo(int ISBN)
+        {
+            IDbConnection db = new SqlConnection("Server = localhost; Database = Bookish; Integrated Security = True; MultipleActiveResultSets = true;");
+            
+            //string sql = "select * from Books WHERE ISBN = " + ISBN;
+            //List<Book> books = (List<Book>)db.Query<Book>(sql);
```

Comment:
Are these commented lines no longer needed? If so you can remove them before merging.